build.gradle

```diff
@@ -211,22 +216,31 @@ tasks.register('aggregatedJavadocs', Javadoc) { aggregated ->
 //  linters for various specific languages or file formats
 //
 
-// Gradle build scripts
-// disabled until <https://github.com/gradle/gradle/issues/6246> is fixed
-/*
+// We use no deprecated Gradle APIs ourselves, and we'd like to keep it that way.  Unfortunately,
+// the Gradle linter tasks produce a deprecation warning under Gradle 7.2.  The warning message
+// is not detailed by default, and could easily mask any new deprecation problems that we might
+// introduce ourselves.  So we don't run the linter by default for now.
```

Comment:
Should this comment go next to the `gradleLint.alwaysRun = false` line?  I think that's what it pertains to

Reply:
Good call. Fixed.